src/ui/material/form-field/src/form-field.wrapper.ts

```diff
@@ -58,13 +58,13 @@ export interface FormlyFieldProps extends CoreFormlyFieldProps {
         >
       </mat-label>
 
-      <ng-container matPrefix *ngIf="props.prefix">
+      <ng-containe matPrefix *ngIf="props.prefix">
```

Comment:
@aitboudad Was this renaming from `ng-container` to `ng-containe` intentional?

Reply:
@Lehoczky not intentional, could you please send a PR. Surprisingly it works :thinking:!

Reply:
Sure thing!